web/src/style-helpers.ts

```diff
@@ -100,3 +95,21 @@ export enum AnimDuration {
   default = "0.3s",
   long = "0.6s",
 }
+
+export const mixinHideOnSmallScreen = `
+@media screen and (max-width: ${Width.smallScreen}px) {
+  display: none;
+}`
+
+export const mixinResetListStyle = `
```

Comment:
should we put these resets in the index.scss stylesheet? i can't think of a case where we want the built-in styles

Reply:
Yeah, good point! Will do.

Reply:
Actually, I'll do that in a separate branch, since it will require changing a bunch of CSS across random components, which will make this PR harder to understand.